website/docs/guides/release-compatibility.html.markdown

```diff
@@ -21,31 +21,31 @@ Quick at-a-glance access to Aviatrix Terraform provider's release compatibility
-| v0.11             | R1.14                     | UserConnect-4.7              |
-| v0.11             | R1.15                     | UserConnect-4.7              |
-| **v0.12 <sup>1</sup>** | R1.16                | UserConnect-4.7              |
-| v0.12             | **R2.0 <sup>2</sup>**     | UserConnect-4.7              |
-| v0.12             | R2.1                      | UserConnect-4.7              |
-| v0.12             | R2.2                      | UserConnect-4.7              |
-| v0.12             | R2.3                      | UserConnect-5.0              |
-| v0.12             | R2.4                      | UserConnect-5.0              |
-| v0.12             | R2.5                      | UserConnect-5.1              |
+| 0.11              | 1.0                      | UserConnect-4.0              |
+| 0.11              | 1.1                      | UserConnect-4.1              |
+| 0.11              | 1.2                      | UserConnect-4.1              |
+| 0.11              | 1.3                      | UserConnect-4.2              |
+| 0.11              | 1.4                      | UserConnect-4.2              |
+| 0.11              | 1.5                      | UserConnect-4.2              |
+| 0.11              | 1.6                      | UserConnect-4.2              |
+| 0.11              | 1.7                      | UserConnect-4.3              |
+| 0.11              | 1.8                      | UserConnect-4.3              |
+| 0.11              | 1.9                      | UserConnect-4.6              |
+| 0.11              | 1.10                     | UserConnect-4.6              |
+| 0.11              | 1.11                     | UserConnect-4.7              |
+| 0.11              | 1.12                     | UserConnect-4.7              |
+| 0.11              | 1.13                     | UserConnect-4.7              |
+| 0.11              | 1.14                     | UserConnect-4.7              |
+| 0.11              | 1.15                     | UserConnect-4.7              |
+| **0.12 <sup>1</sup>** | 1.16                 | UserConnect-4.7              |
+| 0.12              | **2.0 <sup>2</sup>**     | UserConnect-4.7              |
+| 0.12              | 2.1                      | UserConnect-4.7              |
+| 0.12              | 2.2                      | UserConnect-4.7              |
+| 0.12              | 2.3                      | UserConnect-5.0              |
+| 0.12              | 2.4                      | UserConnect-5.0              |
+| 0.12              | 2.5                      | UserConnect-5.1              |
 
 **<sup>1</sup>** : Note that Terraform v0.12 is not backwards-compatible with previous Terraform versions. For R1.16, there will be a need to change some syntax in the Terraform configuration files. Please see Hashicorp's [announcement](https://www.hashicorp.com/blog/announcing-terraform-0-12) for more information
```

Comment:
add an example here too on how to use the version argument

Reply:
Added example under release chart